src/App.tsx

```diff
@@ -1,22 +1,35 @@
-import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import { Auth0Provider } from '@auth0/auth0-react';
+import { CssBaseline, ThemeProvider } from '@mui/material';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
 import { ToastContainer } from 'react-toastify';
-import DashbaordPage from './components/DashboardPage/DashbaordPage';
-import HomePage from './components/HomePage/HomePage';
 import './App.scss';
+import ButtonAppBar from './components/Common/AppBar/AppBar';
+import ProtectedRoute from './components/Common/ProtectedRoute/ProtectedRoute';
+import theme from './themes/theme';
+import DashboardPage from './views/DashboardPage/DashboardPage';
+import HomePage from './views/HomePage/HomePage';
+import NotFoundPage from './views/NotFoundPage/NotFoundPage';
 
 const App = () => {
-
   return (
-    <div className="App" id="app">
-      <BrowserRouter>
-        <Routes>
-          <Route path="/" element={<HomePage />} />
-          <Route path="dashboard" element={<DashbaordPage />} />
-        </Routes>
-      </BrowserRouter>,
-
-      <ToastContainer />
-    </div>
+	<Auth0Provider
+    domain="graphit.us.auth0.com"
```

Comment:
I think I configured my formatter wrong, might need help with that

Reply:
I think you're spaces are on 2 instead of 4, you have in the right-bottom of vscode